apps/schema/src/components/ColumnEditModal.vue

```diff
@@ -252,17 +277,22 @@ export default {
       error: null,
       client: null,
       loading: false,
+      previewShow: false,
+      previewData: {},
     };
   },
   computed: {
     //current table object
     table() {
-      return this.schema.tables.find(
-        (table) =>
-          table.name === this.tableName ||
-          table.name === this.column.table ||
-          (table.subclasses && table.subclasses.includes(this.column.table))
-      );
+      const table = deepClone(this.schema.tables.find(
+          (table) =>
+              table.name === this.tableName ||
+              table.name === this.column.table ||
+              (table.subclasses && table.subclasses.includes(this.column.table))));
+      //replace column with current changes
+      const index = table.columns.findIndex(c => c.name == this.column.name || c.name == this.column.oldName)
```

Comment:
Volgens mij was het in JS altijd handig '===' te gebruiken ipv '=='

Reply:
yeah thx, my problem is switching between java and js ;-)